synapse/tools/rstorm.py

```diff
@@ -56,49 +203,42 @@ async def genStormRst(path, debug=False):
 
             text = line.split('::', 1)[1].strip()
 
-            core = context.get('cortex')
-            if core is None:
-                mesg = 'No cortex set.  Use .. storm-cortex::'
-                raise s_exc.NoSuchVar(mesg=mesg)
+            core = reqCore()
 
-            opts = context.get('opts')
-            await core.callStorm(text, opts=opts)
+            soutp = StormOutput(core, context, stormopts=context.get('storm-opts'))
+            await soutp.runCmdLine(text)
             continue
 
-        if line.startswith('.. storm::'):
+        if line.startswith('.. storm-pkg::'):
+            # load a package into the cortex from a directory
 
-            text = line.split('::', 1)[1].strip()
+            yamlpath = line.split('::', 1)[1].strip()
 
-            core = context.get('cortex')
-            if core is None:
-                mesg = 'No cortex set.  Use .. storm-cortex::'
-                raise s_exc.NoSuchVar(mesg=mesg)
+            core = reqCore()
 
-            outp.append('::\n')
-            outp.append('\n')
+            pkg = s_genpkg.loadPkgProto(yamlpath)
+            await core.addStormPkg(pkg)
+            continue
 
-            outp.append(f'    > {text}\n')
+        if line.startswith('.. storm-mock-http::'):
+            # setup a mock to use with a later storm command
+            jsonpath = line.split('::', 1)[1].strip()
+            context['mock-http'] = s_common.jsload(jsonpath)
```

Comment:
Similar to relative path issues above. We should **fail** here if the path does not exist.

Reply:
ack